File: server/router/cloudpods/network/ip.go
package network

import (
	"github.com/gin-gonic/gin"
	v1 "ygpt/server/api/v1"
)

type NetIpRouter struct {
}

func (s *NetIpRouter) InitNetworkIpRouter(Router *gin.RouterGroup) {
	dataRouter := Router.Group("cloud/network/ip")
	ipApi := v1.ApiGroupApp.CloudApiGroup.NetApiGroup.NetIpApi

	{
		dataRouter.POST("getNetIpList", ipApi.GetNetIPList)         // 获取Ip列表
		dataRouter.POST("addNetIP", ipApi.AddNetIP)                 // 新增Ip
		dataRouter.POST("deleteNetIP", ipApi.DeleteNetIP)           // 删除Ip
		dataRouter.POST("deleteNetIpByIds", ipApi.DeleteNetIpByIds) // 批量删除Ip
		dataRouter.POST("updateNetIP", ipApi.UpdateNetIP)           // 修改Ip
	}
}

File: web/src/api/cloudpods/network/ip.js
import request from '@/utils/request'
//获取
export function getNetIpList(params){
    return request({
        url:'cloud/network/ip/getNetIpList',
        method:'post',
        data: params
    })
}
//新增
export function addNetIP(params){
    return request({
        url:'cloud/network/ip/addNetIP',
        method:'post',
        data: params
    })
}
//删除
export function deleteNetIP(params){
    return request({
        url:'cloud/network/ip/deleteNetIP',
        method:'post',
        data: params
    })
}
//批量删除
export function deleteNetIpByIds(params){
    return request({
        url:'cloud/network/ip/deleteNetIpByIds',
        method:'post',
        data: params
    })
}
//编辑
export function updateNetIP(params){
    return request({
        url:'cloud/network/ip/updateNetIP',
        method:'post',
        data: params
    })
}